File: DataStructure/C#/Stack.cs
using System;
using System.Collections.Generic;
namespace Stack
{
    class MainClass
    {
        public static void Main()
        {
            Stack<int> s = new Stack<int>();

            s.Push(1);
            s.Push(2);
            s.Push(3);
            Console.WriteLine(s.Peek()); // 3  s:[1,2,3]
            s.Push(4);
            Console.WriteLine(s.Peek()); // 4  s:[1,2,3,4]
            s.Pop();
            s.Pop();
            Console.WriteLine(s.Peek()); // 2  s:[1,2]

            return;
        }
    }
}

File: DataStructure/Language/Python/Stack.py
# Pythonの標準ライブラリ
from collections import deque 

# 初期化
box = deque()

# push 要素を一番後ろに追加する
box.append(1)
box.append(2)
box.append(3)
print(box) # deque([1, 2, 3])
box.append(4)
print(box) # deque([1, 2, 3, 4])

# pop 一番後ろ（新しい）要素を取り出し
print(box.pop()) # 4
print(box.pop()) # 3
print(box) # deque([1, 2])